### DISCUSSION TITLE
Any tips on how to fix this other plugin I'm using to play nice with sphinx-autodoc-typehints?

### DISCUSSION THREAD

**Discussion by red8888** [tox-dev/sphinx-autodoc-typehints#303] [Q&A] [answered]
I have functions that are decorated with `@task` (I'm using pyinvoke).

I use this plugin to make autodoc not ignore these annotated fucntions: https://github.com/pyinvoke/invocations/blob/main/invocations/autodoc.py

The issue is that it seems sphinx-autodoc-typehints isn't adding type info to my `@task` functions (but it is doing it for other functions just fine).

I know this is probably more an issue with this other plugin I'm using, FYI I opened an issue asking about it there: https://github.com/pyinvoke/invocations/issues/33

But I'm very new to sphinx and wasn't sure if this is something this plugin would need to support or the other plugin needs to support or if there is a way to configure it to work as is.

Any tips would be super helpful!

**Comment by gaborbernat** [comment, marked as answer]
This is likely not supported, you should add a PR with the feature gaps if you want/need this.
